odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1210,6 +1210,94 @@ Example ##ex:
 ```
 :::
 
+### ##subsubsec Geo Values
+
+CSDL documents with a version of `4.02` or greater MAY use constant values of type `Edm.Geography`, `Edm.Geometry`, or one of their subtypes in annotations.
+
+::: {.varjson .rep}
+Values are represented as GeoJSON, see [OData-JSON](#ODataJSON).
+:::
+
+::: {.varjson .example}
+Example ##ex:
+```json
+"Location": {"type": "Point", "coordinates": [142.1,64.1]}
+```
+:::
+
+::: {.varxml .rep}
+Values are represented as [string expressions](#String) using the WKT (well-known text) format for `Geo` types, see rules
+`fullCollectionLiteral`, `fullLineStringLiteral`,
+`fullMultiPointLiteral`, `fullMultiLineStringLiteral`,
+`fullMultiPolygonLiteral`, `fullPointLiteral`, and
+`fullPolygonLiteral` in
+[OData-ABNF](#ODataABNF).
+:::
+
+::: {.varxml .example}
+Example ##ex:
+```xml
+<PropertyValue Property="Location" String="geography'SRID=0;Point(142.1 64.1)'" />
+```
+:::
+
+### ##subsubsec Stream Values
+
+CSDL documents MAY use constant values of type `Edm.Stream` in annotations.
+
+::: {.varjson .rep}
+Constant values of type `Edm.Stream` are represented according to [OData-JSON](#ODataJSON) and MUST be accompanied by 
+the control information `@mediaContentType` to indicate how the stream value is to be interpreted.
+:::
+
+::: {.varxml .rep}
+Constant values of type `Edm.Stream` with media type `application/json` or one of its subtypes,
```

Comment:
I reviewed this section of section 14.
Previously, it contained the statement to allow constant strings as JSON stream values only in CSDL documents with a version of 4.02 or greater. This is too restrictive, because there are existing examples in version 4.0 that already use this feature. This restriction has been removed now, so the text looks good to me.